primitives/runtime/src/generic/header.rs

```diff
@@ -115,6 +115,21 @@ impl<Number, Hash> Encode for Header<Number, Hash> where
 	}
 }
 
+impl<Number, Hash> scale_info::TypeInfo for Header<Number, Hash> where
+	Number: HasCompact + Copy + Into<U256> + TryFrom<U256> + 'static,
+	Hash: HashT,
+	Hash::Output: scale_info::TypeInfo,
+{
+	type Identity = Self;
+
+	fn type_info() -> scale_info::Type<scale_info::form::MetaForm> {
+		// todo [AJ] provide accurate custom TypeInfo impl
```

Comment:
is this to be fixed before merge ?

Reply:
Missed this one: looking at it now

Reply:
Added a manual impl - please check to confirm

Reply:
yes looks good, out of curiosity, why can't it be automatically derived ?

Reply:
I'd ask the same question about the `Encode` impl.

It could be derived if there was a codec `#[compact]` attribute on `number`.

Reply:
ah I see, let's note it and do it in a follow-up.

Reply:
Noted in PR description

Reply:
Fixed